spec/lib/amazon_product_api/search_response_spec.rb

```diff
@@ -43,4 +43,40 @@
       end
     end
   end
+
+  describe '#success' do
+    context 'a successfull response' do
+      it 'should return true for success codes' do
+        [ 200, 204, 302, 304 ].each do |code|
+          response = AmazonProductAPI::SearchResponse.new({}, code)
+          expect(response.success?).to be(true), "expected true, got #{response.success?} for code #{code}"
+        end
+      end
+
+      it 'should return false for non-success codes' do
+        [ 400, 500 ].each do |code|
```

Comment:
Nit: Similar comment about defining a method for `non_success_codes`?

Reply:
Will do.